src/osd/OSD.cc

```diff
@@ -2142,6 +2142,10 @@ int OSD::mkfs(CephContext *cct,
 	   << "queue_transaction returned " << cpp_strerror(ret) << dendl;
       return ret;
     }
+    C_SaferCond waiter;
+    if (!ch->flush_commit(&waiter)) {
+      waiter.wait();
+    }
```

Comment:
how about
```suggestion
    ch->flush();
```
?

Reply:
Oops, I'm so stupid, sorry for my so late response. fix it soon.

Reply:
good point.